src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -65,6 +67,25 @@ export default function App({ children }) {
     [dispatch, userLoggedIn],
   );
 
+  const CLAIMS_PER_PAGE = 10;
+  let displayedClaims = travelClaims;
+  const shouldPaginate = travelClaims.length > CLAIMS_PER_PAGE;
+  const numPages = Math.ceil(travelClaims.length / CLAIMS_PER_PAGE);
+
+  const pageStart = (currentPage - 1) * CLAIMS_PER_PAGE + 1;
+  const pageEnd = Math.min(currentPage * CLAIMS_PER_PAGE, travelClaims.length);
+
+  if (shouldPaginate) {
```

Comment:
I think we can get rid of this logic right? for example, if `travelClaims.length` is 5, `travelClaims.slice(pageStart - 1, pageEnd)` will yield all 5 claims.